src/reducers/arcReducer.ts

```diff
@@ -184,6 +217,17 @@ export function proposalFailed(proposal: IProposalState) {
   return res;
 }
 
+export function proposalRewardsString(proposal: IProposalState) {
```

Comment:
seems like a UI thing, should not be in the reducer file.

Reply:
yeah, where should it go?

Reply:
`PreTransactionModal.tsx`